lms/product/plugin/grouping.py

```diff
@@ -45,6 +45,9 @@ def get_sections_for_grading(
 
         return None
 
+    def get_group_sets(self, course_id):  # prama: nocover
```

Comment:
```suggestion
    def get_group_sets(self, course_id):  # pragma: nocover
```

But I think you might want an ABC or a protocol, so that coverage knows that these aren't real methods and won't expect them to be covered in the first place.

Reply:
Do we want type annotations on this method? The rest of the methods in this class seem to have them.